instructions on getting a specific holonix version (#421)
holochain/docs-pages: src/pages/get-started/install-advanced.md

nix develop github:holochain/holochain#holonix
```

> To get ad-hoc shell with a specific version of Holochain, use the flag `--override-input versions <version_path>`. This example gives you the newest release in the unstable 0.2.x series:

Author: ```suggestion
To get an ad-hoc shell with a specific version of Holochain, use the flag `--override-input versions <version_path>`. This example gives you the newest release in the unstable 0.2.x series:
```

```shell

> nix develop --override-input versions "github:holochain/holochain?dir=versions/0_2" "github:holochain/holochain#holonix"

Author: ```suggestion
nix develop --override-input versions "github:holochain/holochain?dir=versions/0_1" "github:holochain/holochain#holonix"
```

```

> And this example gives you the newest weekly developer snapshot:

Author: ```suggestion
And this example gives you the newest weekly developer snapshot (currently in the 0.3.x series):
```

```shell
nix develop --override-input versions ./versions/weekly .#holonix
```

> Take a look at the [`versions/` folder in the `holochain/holochain` repository](https://github.com/holochain/holochain/tree/develop/versions) to find out what versions you can target. Each subfolder is a valid version number to use in the above command, and the `flake.nix` file inside the subfolder shows the specific release tag for each component that will be downloaded.

Reviewer: ```suggestion
Take a look at the [`versions/` folder in the `holochain/holochain` repository](https://github.com/holochain/holochain/tree/develop/versions) to find out what versions you can target. Each sub-directory is a valid version to use in the above command, and the `flake.nix` file inside the sub-directory shows the specific release tags that will be used for Holochain and the Lair Keystore. For the launcher and scaffolding, you will get the latest release of those tools that are available on the release branch referenced in the `flake.nix`. Note that you may temporarily see older versions than the `flake.nix` suggests because our automation updates the `flake.lock` separately, so that file is the absolute source of truth for what versions you will get.
```

Reviewer: This is wordy but contains more details. Up to you what you want to do with it! Maybe you can shorten it a bit while keeping the important details?

Author: Wordy is good. This is a page for devs who are looking for details, so I think we should keep it all in. I'm a bit torn on subfolder vs subdirectory though; I've used 'folder' everywhere else in the documentation. Thoughts on whether 'directory' is an important distinction to make?

Reviewer: Not important, happy to drop that from my suggestion

Reviewer: Done

Author: @ThetaSinner I've edited it a bit for clarity; did I get the details right? As I understand this, launcher and scaffolding have branches that maintain compatibility with a Holochain major release series, but the `flake.lock` might cause Nix to pull in a slightly older commit on that branch if CI hasn't refreshed `flake.lock` lately?

Reviewer: Yes I think that is all accurate and clear but because of the ordering it might sound like only scaffolding and launcher can be behind. The same is also true of Holochain and Lair Keystore. If I update the `flake.nix` to use the `0.2.6` tag for Holochain but the `flake.lock` will still be pointing at `0.2.5` until the automation bumps it. It runs every 6 hours currently so it's not a long lag but if the CI fails then it can run a few days behind until we get it fixed.
